validator/client/validator_test.go

```diff
@@ -1315,10 +1315,10 @@ func TestValidator_WaitForKeymanagerInitialization_Web(t *testing.T) {
 	require.NoError(t, err)
 	walletChan := make(chan *wallet.Wallet, 1)
 	v := validator{
-		db:                       db,
-		useWeb:                   true,
-		walletInitializedFeed:    &event.Feed{},
-		walletInitializedChannel: walletChan,
+		db:                    db,
+		useWeb:                true,
+		walletInitializedFeed: &event.Feed{},
+		walletInitializedChan: walletChan,
```

Comment:
I wonder if we should use Chan or Ch ? maybe we should choose one way of writing it to be consistent